sh/tests/solutions/calculator.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+number='^-?[0-9]+$'
+
+do_add () {
+    echo $(($1 + $2))
+}
+
+do_sub () {
+    echo $(($1 - $2))
+}
+
+do_mult () {
+    echo $(($1 * $2))
+}
+
+do_divide () {
+    echo $(($1 / $2))
+}
+
+
+if [ $# != 3 ]
+then
+    >&2 echo "Error: expect 3 arguments"
+    exit 1
+elif ! [[ $1 =~ $number && $3 =~ $number ]]
+then
+    >&2 echo "Error: not a valid number"
```

Comment:
different message in the readme
`"Error: invalid number"`

Reply:
updated the solution to match the subject